content/en/blog/2025/kubecon-eu.md

```diff
@@ -0,0 +1,222 @@
+  17:45 - 18:15
+- **[OTel Sucks (But Also Rocks!)](https://sched.co/1txHm)**<br> by Juraci
+  Paixão Kröhling, OllyGarden & Daniel Dyla, Dynatrace<br> Thursday, April 3 •
+  11:00 - 11:30
+- **[Wasm Whiplash: WasmCloud's Wild Ride To Standards](https://sched.co/1tcz9)**<br>
+  by Brooks Townsend, Cosmonic<br> Thursday, April 3 • 11:45 - 12:15
+- **[How To Adopt OpenTelemetry in an Enterprise Where Incumbent Vendor Tools Reign Supreme](https://sched.co/1txHs)**<br>
+  by Chris Weldon, Wolters Kluwer<br> Thursday, April 3 • 14:15 - 14:45
+- **[How To Rename Metrics Without Impacting Somebody’s Observability](https://sched.co/1txHv)**<br>
+  by Bartłomiej Płotka, Google & Arianna Vespri, Independent<br> Thursday, April
+  3 • 15:00 - 15:30
+- **[OTel Me How To Get My Open Source Community Taken Seriously: Lessons Learned as an OTel Maintainer](https://sched.co/1txH6)**<br>
+  by Reese Lee, New Relic & Adriana Villela, Dynatrace<br> Thursday, April 3 •
+  16:45 - 17:15
+- **[⚡ Lightning Talk: Observability Diet: Your 5-Step Plan To Trim the Data Fat](https://sched.co/1txCq)**<br>
+  by Pranay Prateek, SigNoz<br> Thursday, April 3 • 16:55 - 17:00
+- **[Optimizing Metrics Collection & Serving When Autoscaling LLM Workloads](https://sched.co/1txI4)**<br>
+  by Vincent Hou, Bloomberg & Jiří Kremser, kedify.io<br> Thursday, April 3 •
+  17:30 - 18:00
+- **[The State of Prometheus and OpenTelemetry Interoperability](https://sched.co/1txIA)**<br>
+  by Arthur Sens, Grafana & Juraj Michálek, Swiss RE<br> Friday, April 4 •
+  11:45 - 12:15
+- **[Fluent Bit v4: A Decade of Innovation and What’s Next](https://sched.co/1ue2s)**<br>
+  by Eduardo Silva, Chronosphere<br> Friday, April 4 • 13:45 - 14:15
+- **[Smooth Scaling With the OpAMP Supervisor: Managing Thousands of OpenTelemetry Collectors](https://sched.co/1txID)**<br>
+  by Evan Bradley, Dynatrace & Andy Keller, observIQ<br> Friday, April 4 •
+  13:45 - 14:15
+- **[Evolving OpenID Connect and Observability in Keycloak](https://sched.co/1td1c)**<br>
+  by Ryan Emerson, Red Hat & Takashi Norimatsu, Hitachi<br> Friday, April 4 •
+  14:30 - 15:00
+- **[How To Supercharge AI/ML Observability With OpenTelemetry and Fluent Bit](https://sched.co/1txAi)**<br>
+  by Celalettin Calis, Chronosphere<br> Friday, April 4 • 15:15 - 15:45
+
+## Observability Day
+
+[Observability Day] fosters collaboration, discussion, and knowledge sharing of
+cloud native observability projects. This event will be held on Tuesday, April
+1, 2025, from 9:00 - 17:25. There will be several sessions on OpenTelemetry as
+well.
+
+{{% alert title="Important access note" color="danger" %}}
+
+You need an _all-access_ pass to attend **Observability Day**. For details, see
+[KubeCon registration][registration].
+
+[registration]:
+  https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/?utm_source=opentelemetry&utm_medium=all&utm_campaign=KubeCon-EU-2025&utm_content=blog
+
+{{% /alert %}}
```

Comment:
The `[registration][]` in the first paragraph isn't resolving:

> <img width="629" alt="image" src="https://github.com/user-attachments/assets/43bd740d-5a82-4b87-9f78-0c68c4defb5c" />

This should fix it:

```suggestion
{{% alert-md title="Important access note" color="danger" %}}

You need an _all-access_ pass to attend **Observability Day**. For details, see
[KubeCon registration][registration].

[registration]:
  https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/?utm_source=opentelemetry&utm_medium=all&utm_campaign=KubeCon-EU-2025&utm_content=blog

{{% /alert-md %}}
```

Reply:
Yes, that worked:

> <img width="615" alt="image" src="https://github.com/user-attachments/assets/d0da2d43-7207-4743-b33f-03c1ef0f8ac8" />